src/resolver.ts

```diff
@@ -3362,4 +3341,24 @@ export class Resolver extends DiagnosticEmitter {
     }
     return instance;
   }
+
+  private ensureOneTypeArgument(
+    /** The type to resolve. */
+    node: NamedTypeNode,
+    /** How to proceed with eventual diagnostics. */
+    reportMode: ReportMode = ReportMode.REPORT): TypeNode | null { 
```

Comment:
```suggestion
  private ensureOneTypeArgument(
    /** The type to resolve. */
    node: NamedTypeNode,
    /** How to proceed with eventual diagnostics. */
    reportMode: ReportMode = ReportMode.REPORT
  ): TypeNode | null { 
```

Reply:
Should this have been caught by the format check?

Reply:
I don't know, but this general style convention